config/quality/checkstyle/checkstyle.xml

```diff
@@ -50,7 +50,9 @@
         <module name="EmptyStatement" />
         <module name="EqualsAvoidNull" />
         <module name="EqualsHashCode" />
-        <module name="ExplicitInitialization" />
+        <module name="ExplicitInitialization">
```

Comment:
Because you can argue that `boolean foobar = false` or that `int baz = 0` is more clear and more explicit than just `boolean foobar` or `int baz` while it has no effect on the performance (the generated bytecode is exactly the same)